apps/shared/WebView.qml

```diff
@@ -106,10 +106,18 @@ WebContainer {
     fullscreenMode: (contentItem && !contentItem.chrome) ||
                     (contentItem && contentItem.fullscreen)
 
+    selectionActive: webView.contentItem && webView.contentItem.textSelectionActive
     touchBlocked: contentItem && contentItem.popupOpener && contentItem.popupOpener.active ||
-                  webView.contentItem && webView.contentItem.textSelectionActive || !AccessPolicy.browserEnabled || false
+                   selectionActive || !AccessPolicy.browserEnabled || false
     favicon: contentItem ? contentItem.favicon : ""
 
+    onSelectionActiveChanged: {
+        if (!selectionActive) {
```

Comment:
After https://github.com/sailfishos/sailfish-browser/pull/982/files#r798434399 this can call clearSelection of the contentItem. Also please guard against null contentItem.
```
onSelectionActiveChanged: {
    if (!selectionActive && contentItem) {
        webView.contentItem.clearSelection()
    }
}
```